modules/core/src/main/java/org/apache/ignite/configuration/DataStorageConfiguration.java

```diff
@@ -601,6 +621,19 @@ public DataStorageConfiguration setExtraStoragePaths(String... extraStoragePaths
         return this;
     }
 
+    /**
+     * Sets a paths to the root directories where the snapshot files stored.
+     * By default, {@link IgniteConfiguration#getSnapshotPath()} used.
+     *
+     * @param extraSnapshotPaths Extra snapshot paths where snapshot files can be stored.
+     * @return {@code this} for chaining.
+     */
+    public DataStorageConfiguration setExtraSnapshotPaths(String... extraSnapshotPaths) {
```

Comment:
Let's mention in docs that amount of extraSnapshotPaths must be equal amount of extra storage paths

Reply:
Done